pom.xml

```diff
@@ -50,7 +50,32 @@
     </pluginRepository>
   </pluginRepositories>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jenkins-ci.main</groupId>
+        <artifactId>jenkins-bom</artifactId>
+        <version>${jenkins.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
   <dependencies>
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-jaxrs</artifactId>
+      <version>4.0.0.Beta5</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.ant</groupId>
+      <artifactId>ant</artifactId>
```

Comment:
Why is this dependency needed here?

Reply:
It _isn't_ needed. That was the point in the PR text ^^,  that I just added some fake dependencies to illustrate bom usage cf [PR-131](https://github.com/jenkinsci/cloudbees-folder-plugin/pull/130)

Reply:
Uh, OK. This plugin is probably not a helpful example of core BOM usage because it does not use any special libraries. The only cases that I know of where you would benefit from importing the core BOM would be a plugin that bundles a library (dep on non-plugin → `WEB-INF/lib/*.jar`) which is _not_ using `pluginFirstClassLoader` and you would otherwise run into conflicts between a transitive dependency of that library and something which is actually accessible via `jenkins-core.jar`. That is why I suggested looking for something that depends on a library that depends on `commons-io` or similar.

Reply:
Yeah, true - it was a contrived example with fake dependencies just to illustrate the point.
To your `commons-io` suggestion: 'blueocean-display-url-plugin' depends on `commons-codec` which results on enforcer errors on both slf4j & commons-codec when building against 2.195 so I re-did the example there:
* [With BOM](https://github.com/jenkinsci/blueocean-display-url-plugin/pull/46)
* [Without BOM](https://github.com/jenkinsci/blueocean-display-url-plugin/pull/47)
HTH